utils/flags/flags.go

```diff
@@ -338,6 +338,12 @@ func GetCliFlags() []cli.Flag {
 			Usage:   "The S3/minio secret access key to use when using S3 proxy backend. " + s3AuthMsg(s3proxy.AuthMethodAccessKey),
 			EnvVars: []string{"BAZEL_REMOTE_S3_SECRET_ACCESS_KEY"},
 		},
+		&cli.StringFlag{
+			Name:    "s3.session_token",
+			Value:   "",
+			Usage:   "The S3/minio session token to use when using S3 proxy backend. Optional. " + s3AuthMsg(s3proxy.AuthMethodAccessKey),
+			EnvVars: []string{"BAZEL_REMOTE_S3_SESSION_TOKEN", "AWS_SESSION_TOKEN"},
```

Comment:
I wonder if we could skip the AWS_SESSION_TOKEN variable here- we don't make use of any of the other environment variables mentioned in https://docs.aws.amazon.com/cli/latest/topic/config-vars.html#credentials

Reply:
Ah, yes, I see: I was keying off the reading of `AWS_SHARED_CREDENTIALS_FILE` and `AWS_PROFILE` below, but indeed the `--s3.access_key_id` and `--s3.secret_access_key` args support the corresponding `AWS_...` variable. The last two there are more closely related to this work, so better to follow them!

I've remove that from this PR, and it can be a separate bit of work (if at all) to read those AWS ones.

Thanks for catching 😄